custom_components/trakt_tv/schema.py

```diff
@@ -76,4 +77,32 @@ def recommendation_schema() -> Dict[str, Any]:
     return subschemas
 
 
+def stats_schema() -> list[str]:
+    return [
```

Comment:
I had to use chatGPT because the documentation is definitely not clear... It seems it should be like: ` "stats": All(Any(["a", "b", "c"]), str))` can you try the schema validation using this rule by putting false information in your YAML file ? Maybe you already tried and your code is already OK.

Reply:
I'd say documentation is more towards _non-existant_ rather than unclear 😃 
Just Schema works fine
```
2024-06-04 16:38:49.072 ERROR (MainThread) [homeassistant.config] Invalid config for 'trakt_tv' at configuration.yaml, line 30: not a valid value 'trakt_tv->sensors->stats->0', got 'abc', please check the docs at https://github.com/dylandoamaral/trakt-integration
```

Reply:
Ok perfect !